week-1/mandatory/2-classes-db/cyf_classes.sql

```diff
@@ -66,5 +66,6 @@ SELECT * FROM mentors WHERE years_in_glasgow > 5;
 SELECT * FROM mentors WHERE fav_programming_lang = 'Java';
 SELECT * FROM students WHERE cyf_graduate = 'true';
 SELECT * FROM classes WHERE class_date < '2020-06-01';
-SELECT * FROM attendance WHERE topic = 'NodeJS';
-SELECT * FROM classes WHERE topic = 'Javascript';
\ No newline at end of file
+SELECT * FROM attendance WHERE topic = 'Javascript'; 
+-- OR
+SELECT student_id, classes.topic FROM attendance            INNER JOIN classes ON attendance.class_id = classes.id WHERE classes.topic = 'Javascript';
```

Comment:
I think this is the one we went over on the call, but in this case the second answer is probably best, while there are sometimes reasons to, in general its best to try to only define data in one place and so the topic field probably shouldnt be part of the attendance table.